ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -262,6 +262,16 @@ CO2C:
   - zone_air_co2_concentration_sensor
   - zone_air_co2_concentration_setpoint
 
+CO2CDSP:
+  description: "Carbon dioxide control with dual setpoint."
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  uses:
+  - high_limit_zone_air_co2_concentration_setpoint
+  - low_limit_zone_air_co2_concentration_setpoint
```

Comment:
What does it do if the CO2 is below setpoint? This point seems irrelevant, given the device cannot do anything if the value falls below the setpoint. 

Seems like the device would control like a normal `CO2C`-typed device. What am I missing?

Reply:
Hello,

The both setpoints are used for the regulation. If you go below the low_limit or above the high_limit, nothing special happens. However, in between, the damper's opening not the same. I attached a drawing to illustrate the regluation with 700 ppm as low_limit setpoint and 1100 ppm as high_limit setpoint.
![IMG_20230130_135037](https://user-images.githubusercontent.com/83808569/215482450-3f9f79a5-2a30-4825-a86f-3770fa5154d4.jpg)